sys/linux/dev_vga_arbiter.txt

```diff
@@ -0,0 +1,44 @@
+# Copyright 2020 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+resource fd_vga_arbiter[fd]
+openat$vga_arbiter(fd const[AT_FDCWD], file ptr[in, string["/dev/vga_arbiter"]], flags flags[open_flags], mode const[0]) fd_vga_arbiter
+
+write$vga_arbiter(fd fd_vga_arbiter, data ptr[in, data_vga_arbiter], len bytesize[data])
+
+data_vga_arbiter [
+	target_default	string["target default"]
+	unlock_all	string["unlock all"]
```

Comment:
Unlock all seems like an odd choice to hard-code, since https://www.kernel.org/doc/html/latest/gpu/vgaarbiter.html says it's not implemented yet?

Also reading vga_arb_write, won't this always go down the `else if (strncmp(curr_pos, "target ", 7) == 0)` branch and ignore the rest of the input after "target default"?

Reply:
`unlock all` is actually implemented https://elixir.bootlin.com/linux/v5.8/source/drivers/gpu/vga/vgaarb.c#L1134. I think the document hasn't been updated.

Regarding `target default`, this is an enum so it will choice one from the four variants `target_default`, `unlock_all`, `target`, `other`, so I think this won't happen?

Reply:
Ahh, derp, my bad, I mistook it for a struct